hadoop-yarn-project/hadoop-yarn/hadoop-yarn-csi/src/test/java/org/apache/hadoop/yarn/csi/client/TestCsiClient.java

```diff
@@ -37,15 +37,14 @@
 public class TestCsiClient {
 
   private static File testRoot = null;
+  private static File socketFile = null;
   private static String domainSocket = null;
   private static FakeCsiDriver driver = null;
 
   @BeforeClass
   public static void setUp() throws IOException {
-    testRoot = GenericTestUtils.getTestDir("csi-test");
-    File socketPath = new File(testRoot, "csi.sock");
-    FileUtils.forceMkdirParent(socketPath);
-    domainSocket = "unix://" + socketPath.getAbsolutePath();
+    socketFile = new File("/tmp", "yarn-csi-test.sock");
```

Comment:
Please take care of deleting this file in teardown().

Reply:
@vinayakumarb ,thanks for your review, done